eclair-core/src/main/scala/fr/acinq/eclair/wire/ChannelCodecs.scala

```diff
@@ -126,19 +133,10 @@ object ChannelCodecs extends Logging {
     .typecase(0x09, (("inputInfo" | inputInfoCodec) :: ("tx" | txCodec)).as[HtlcPenaltyTx])
     .typecase(0x10, (("inputInfo" | inputInfoCodec) :: ("tx" | txCodec)).as[ClosingTx])
 
-  // this is a backward compatible codec (we used to store the sig as DER encoded), now we store it as 64-bytes
-  val sig64OrDERCodec: Codec[ByteVector64] = Codec[ByteVector64](
```

Comment:
Got rid of this compatibility code.

Reply:
We will indeed always read with the legacy backward-compatible codec and write with the new codec. There is no need for the new codec to support legacy encoded data.